src/utils/executors.ts

```diff
@@ -0,0 +1,26 @@
+import {Executor} from '@models/executors';
+import {Option} from '@models/form';
+
+export const remapExecutors = (executors: Executor[]): Option[] => {
+  if (!executors || !executors.length) {
+    return [];
+  }
+
+  const array: Option[] = [];
+
+  executors.forEach(executorItem => {
+    const {
+      executor: {types, executorType},
+    } = executorItem;
+
+    if (types) {
+      types.forEach(type => {
+        array.push({label: type, value: type});
+      });
```

Comment:
What do you think about i.e.:

```suggestion
      array.push(...types.map(type => ({label: type, value: type})));
```

And similarly below?